app/src/main/java/android/support/v17/leanback/supportleanbackshowcase/app/media/VideoMediaPlayerGlue.java

```diff
@@ -254,6 +383,37 @@ public void releaseResources() {
         }
     }
 
+    public void createMediaSessionIfNeeded() {
+        if (mVideoSession == null) {
+            mVideoSession = new MediaSession(this.getContext(), "VideoPlayer Session");
```

Comment:
Well, I don't think we want to destroy and recreate the session on configuration change, so releasing it in onStop doesn't make sense.

My reading of the API is that the MediaSession should live in the playback service, and the UI should talk to it by way of a MediaController.